cygnus-ngsi/src/test/java/com/telefonica/iot/cygnus/sinks/NGSICKANSinkTest.java

```diff
@@ -1026,10 +1054,50 @@ public void testBuildResourceNameLength() throws Exception {
         } // try catch
     } // testBuildResourceNameLength
     
+    /**
+     * [NGSICKANSink.configure] -------- pkg_by_entity can only be 'true' or 'false'.
+     */
+    @Test
+    public void testPkgByEntity() {
+        System.out.println(getTestTraceHead("[NGSICKANSink.configure]")
+                + "-------- pkg_by_entity can only be 'true' or 'false'");
+        String apiKey = null; // default
+        String attrPersistence = null; // default
+        String backendMaxConns = null; // default
+        String backendMaxConnsPerRoute = null; // default
+        String batchSize = null; // default
+        String batchTime = null; // default
+        String batchTTL = null; // default
+        String dataModel = null; // default
+        String enableEncoding = null;//default
+        String enableGrouping = null; // default
+        String enableLowercase = null; // default
+        String host = null; // default
+        String port = null; // default
+        String ssl = null; // default
+        String viewer = null; // default
+        String confOrganization = "false"; //default
+        NGSICKANSink sink = new NGSICKANSink();
+        sink.configure(createContext(apiKey, attrPersistence, backendMaxConns, backendMaxConnsPerRoute, batchSize,
+                batchTime, batchTTL, dataModel, enableEncoding, enableGrouping, enableLowercase, host, port, ssl,
+                viewer,confOrganization));
+        
+        try {
+            assertTrue(!sink.getInvalidConfiguration());
```

Comment:
Not very convinced about this test. This test doesn't actually test the new implementation or new code at all.

Reply:
Fixed in 6a8704233ea63853e822d9868a6f93b429ea60d7